resources/sass/_temp.scss

```diff
@@ -50,6 +50,10 @@ a:hover {
     @apply text-white;
 }
 
+.full-height {
+    height: 100%;
+}
+
```

Comment:
Hi @umangbp!

Thanks for completing this task so quickly. We use [TailwindCSS](https://tailwindcss.com/docs/height) framework for adding styles to elements in this project.

They actually have a built-in class that we can use here: `h-full`. Would you mind updating the PR to use this class instead? Thank you! 😄 
```suggestion
```

Reply:
Thanks for flagging that. I will change it.